source/dub/dub.d

```diff
@@ -247,6 +247,75 @@ class Dub {
 		m_project = new Project(m_packageManager, pack);
 	}
 
+	/** Loads a single file package.
+
+		Single-file packages are D files that contain a package receipe comment
```

Comment:
Hmm, I would have expected github to recognize .editorconfig and to use 4 space spaces for tabs. Anyway, not important.